backend/main.py

```diff
@@ -10,61 +10,92 @@
 @recipeAPI.route('/api/addRecipe', methods=['POST'])
 def create_recipe():
     data = request.get_json()
+    id = data['id']
     name = data['name']
     ingredients = data['ingredients']
+    measurements = data['measurements']
+    steps = data['steps']
 
-    recipe = RecipeBook.Recipe(name, ingredients)
-    recipeBook.add_recipe(recipe)
+    recipe = Recipe.Recipe(name, ingredients,measurements, steps)
+    recipes = [recipe]
+    Recipe.addRecipe(id, recipes)
     return jsonify({'message': 'Recipe created successfully'})
 
 
 @recipeAPI.route('/api/removeRecipe', methods=['POST'])
 def delete_recipe():
     data = request.get_json()
+    id = data['id']
     name = data['name']
     ingredients = data['ingredients']
+    measurements = data['measurements']
+    steps = data['steps']
 
-    recipe = RecipeBook.Recipe(name, ingredients)
-    recipeBook.remove_recipe(recipe)
+    recipe = Recipe.Recipe(name, ingredients, measurements, steps)
+    name = [recipe]
+    Recipe.removeRecipe(id, name)
     return jsonify({'message': 'Recipe deleted successfully'})
 
 
-@recipeAPI.route('/api/getRecipes', methods=['GET'])
+@recipeAPI.route('/api/getRecipes', methods=['POST'])
 def get_recipes():
-    recipes = recipeBook.get_recipes()
-    recipe_data = [{'name': recipe.get_name(), 'ingredients': recipe.get_ingredients()} for recipe in recipes]
-    return jsonify(recipe_data)
+    data = request.get_json()
+    id = data['id']
+    return Recipe.getRecipes(id)
+    
 
 
 @recipeAPI.route('/api/addIngredient', methods=['POST'])
```

Comment:
change route to include either stock or grocery list so we know what we are adding ingredient to